### DISCUSSION TITLE
Is it possible to decompile from just some packages or a specific package?

### DISCUSSION THREAD

**Discussion by tivrfoa** [Vineflower/vineflower#174] [Q&A] [answered]
Hi!

Is it possible to decompile from just some packages or a specific package, please?

**Comment by jaskarth** [comment, marked as answer]
Hey, that's not implemented functionality at the moment but I'll put in an issue so that it can be implemented in the future :)

**Reply by tivrfoa** [reply]
Thank you! =)
